app/src/main/java/ch/admin/bag/dp3t/networking/ConfigWorker.java

```diff
@@ -44,7 +45,9 @@
 
 	public static void scheduleConfigWorkerIfOutdated(Context context) {
 		if (SecureStorage.getInstance(context).getLastConfigLoadSuccess() <
-				System.currentTimeMillis() - MAX_AGE_OF_CONFIG_FOR_RELOAD_AT_APP_START) {
+				System.currentTimeMillis() - MAX_AGE_OF_CONFIG_FOR_RELOAD_AT_APP_START ||
+				SecureStorage.getInstance(context).getLastConfigLoadSuccessAppVersion() != BuildConfig.VERSION_CODE ||
+				SecureStorage.getInstance(context).getLastConfigLoadSuccessSdkInt() != Build.VERSION.SDK_INT) {
```

Comment:
3x `SecureStorage.getInstance(context)` in one statement; could use a local variable instead.